ibis/backends/flink/tests/test_translator.py

```diff
@@ -0,0 +1,9 @@
+import ibis
+from ibis.backends.flink.compiler.core import translate
+
+
+def test_translate_toplevel(snapshot):
+    t = ibis.table([('foo', 'double')], name='t0')
+    expr = t.foo.sum()
+    result = translate(expr.as_table().op())
+    snapshot.assert_match(str(result), "out.sql")
```

Comment:
It'd be good to add some more tests in `test_translator.py` to cover all the operations you defined in `registry.py`. Right now most of that file has no coverage.